src/layout.c

```diff
@@ -2347,6 +2347,8 @@ int igraph_layout_mds(const igraph_t* graph, igraph_matrix_t *res,
     igraph_matrix_t m;
     igraph_bool_t conn;
 
+    IGRAPH_UNUSED(options);
+
```

Comment:
The "options" argument should be removed IMHO. It used to be used when `igraph_layout_mds` used ARPACK to calculate eigenvectors, but it was a bad decision anyway because the matrices that MDS works on are not sparse. Currently the documentation says that this argument is ununsed, and since we are breaking the API anyway for 0.9, we can remove the argument as I see zero chance that it gets re-introduced in the future.

Reply:
I would be fine with removing. The reason that I left it in is because I couldn't foresee whether this argument would be useful at some point when we would like to change the implementation. To me it sounds reasonable that we will never use `ARPACK` for this, so it might make sense to remove it then indeed.

Reply:
Okay, let's remove it then!